decidim-proposals/spec/lib/decidim/proposals/component_spec.rb

```diff
@@ -148,6 +148,32 @@
       end
     end
 
+    context "when support limit per participant is empty" do
+      before do
+        visit edit_component_path
+        component.update(settings: { vote_limit: "" })
+        visit edit_component_path
+      end
+
+      it "does NOT allow updating proposal component" do
+        click_button "Update"
+        expect(page).to have_content("There's an error in this field")
+      end
+    end
+
+    context "when minimum supports per user is empty" do
+      before do
+        visit edit_component_path
+        component.update(settings: { minimum_votes_per_user: "" })
+        visit edit_component_path
+      end
+
+      it "does NOT allow updating proposal component" do
```

Comment:
Same as my other comment: 
```suggestion
      it "does not allow updating proposal component" do
```